scripts/seeders/library_ingestion.py

```
@@ -81,14 +92,58 @@ def _load_config(self) -> Dict[str, Any]:
+                
+                if ext['is_rom']:
+                    supported['rom'].append(ext_name)
+                if ext['is_archive']:
+                    supported['archive'].append(ext_name)
+                if ext['is_save']:
+                    supported['save'].append(ext_name)
+                if ext['is_patch']:
+                    supported['patch'].append(ext_name)
+            
+            self.logger.info(f"Loaded {len(extensions)} supported extensions from registry")
+            return supported
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)